tests/out/stop_times.txt

```diff
@@ -0,0 +1 @@
+trip_id,arrival_time,departure_time,stop_id,stop_sequence
```

Comment:
@mehdisadeghi is this suppose to be empty? I think it was full some commits ago 🤔

Reply:
> some commit ago

:D oh man... I have to take a break ;-)

Reply:
hahahha yeah I agree. The rest looks pretty good so just to prevent making this PR even bigger I would say to stop there.

Reply:
@carocad I found the problem. I'll push a fix in a few minutes, I'm checking that. While fixing that I found a case related to multi-regional routes which you can find in the lengthy comment inside `gtfs_dummy.py` file.